Makefile

```diff
@@ -54,6 +54,9 @@ init: node_modules ;
 # Use building English language as a check to see if everything works
 test: en ;
 
+lint:
```

Comment:
:+1: on this. @nylira how do you feel about it? (Since you're the one that ends up building and publishing all the changes everytime for the actual site :wink:)